app/controllers/news_controller.rb

```diff
@@ -2,14 +2,14 @@ class NewsController < ApplicationController
 
   def index
     @title = "News"
-    @new_genotypes = Genotype.all(:order => "created_at DESC", :limit => 20)
-    @new_users = User.all(:order => "created_at DESC", :limit => 20)
-    @new_phenotypes = Phenotype.all(:order => "created_at DESC", :limit => 20)
-    @new_phenotype_comments = PhenotypeComment.all(:order => "created_at DESC", :limit => 20)
-    @new_snp_comments = SnpComment.all(:order => "created_at DESC", :limit => 20)
+    @new_genotypes = Genotype.order('created_at DESC').limit(20)
+    @new_users = User.order('created_at DESC').limit(20)
+    @new_phenotypes = Phenotype.order('created_at DESC').limit(20)
+    @new_phenotype_comments = PhenotypeComment.order('created_at DESC').limit(20)
```

Comment:
Line is too long. [81/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.